docs/installing.md

```diff
@@ -1,133 +1,26 @@
 # Installation
 
-During the setup, we'll deploy Tracetest and Postgres with Helm.
+## TL;DR
```

Comment:
Not sure what this is so I just wanted to check that it is what you want the heading to say.

Reply:
Yes, it's short for "Too Long;Didn't Read": https://www.merriam-webster.com/dictionary/TL%3BDR
Very common in docs to show a quick copy/paste cmd without getting into the details